### PR TITLE
Feature desktop final

### PR DESCRIPTION
**Pull request by somarib**
Concerted mobile version into a desktop version of my portfolio using responsive design

### CODE REVIEW

File: stylesheet.css [hunk 1]
```diff
@@ -373,3 +394,493 @@ a {
...
+    align-items: flex-start;
+    width: 40%;
+    height: 100%;
+    margin-top: 20px;
+  }
+
+  .hh2 {
+    width: 327px;
+    font-size: 30px;
+    font-weight: 500;
```
Comment (ichala) [inline stylesheet.css]: - [ ] kindly update the font weight for the project title to `700 ` to match Figma design 


Figma             |  Your Design
:-------------------------:|:-------------------------:
image  |  image

File: stylesheet.css [hunk 2]
```diff
@@ -373,3 +394,493 @@ a {
...
+  .seeproject {
+    display: inline-block;
+    color: #edecff;
+    width: 89px;
+    height: 50px;
+    background-color: #7d73fc;
+    margin-top: 80px;
+    line-height: 48px;
+    text-align: center;
+  }
```
Comment (ichala) [inline stylesheet.css]: - [ ] kindly try to apply the left and right paddings for your buttons to match Figma design

Figma           |  Your Design
:-------------------------:|:-------------------------:
image |  image

File: stylesheet.css [hunk 3]
```diff
@@ -373,3 +394,493 @@ a {
...
+
+  .myform .dsk {
+    display: block;
+    width: 100px;
+    height: 50px;
+    background-color: #7d73fc;
+    border-style: none;
+    transition: width 3s, backgeround-color 4s;
+  }
```
Comment (ichala) [inline stylesheet.css]: - [ ] kindly fix the style of the contact button to match Figma

- [ ] [OPTIONAL] I would suggest that you add a `cursor: pointer;` style to your  button as well to change the cursor type when users hover on the buttons, that way your buttons will be more elegant and user friendly

Figma            |  Your Design
:-------------------------:|:-------------------------:
image  |  image

File: stylesheet.css [hunk 4]
```diff
@@ -373,3 +394,493 @@ a {
...
+    width: 60%;
+    height: 48px;
+    margin-bottom: 15px;
+  }
+
+  .myform textarea {
+    width: 60%;
+    height: 148px;
+    margin-bottom: 15px;
+  }
```
Comment (ichala) [inline stylesheet.css]: - [ ] kindly add the paddings for your inputs like shown in Figma 

Solarized dark             |  Solarized Ocean
:-------------------------:|:-------------------------:
image  |  image

**Review by ichala — CHANGES_REQUESTED** [review]
Hi @somarib , 

Good job so far! 👏👏👏 
There are some issues that you still need to work on to go to the next project but you're almost there!

the-simpsons-homer-simpson

## To Highlight 🎉

- Used Media Query✅ 
- Great Design ✅ 

 ## Required Changes ♻️

- [ ] _**check the comments below**_


## Optional suggestions

N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by zainsadaqat — APPROVED** [review]
# APPROVED!!!! 🎉 🎉 🎉 🎉 🎉 🎉
## Hey @somarib  😊 😃 

#### Great Job 😄 👏 You've done it. Your project is complete!  There is nothing else to say other than... it's time to merge it 👍🏻 ✅ 

# Congratulations!!! 🎉 🥳 👏 🙌 🎊 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**